smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/CreatePageAsAuthorUserIT.java

```diff
@@ -109,4 +117,49 @@ public void testCreatePageAsAuthor() throws InterruptedException, ClientExceptio
         }
     }
 
+    private SlingHttpResponse createTestPage(String pageName, int authRetries) throws ClientException, InterruptedException {
+        Supplier<SlingClient> creatorSupplier = cqBaseClassRule.authorRule::getAdminClient;
+        CQSecurityClient securityClient = creatorSupplier.get().adaptTo(CQSecurityClient.class);
+        CQPermissions permissionsObj = new CQPermissions(securityClient);
+
+        // check if the parent path of the test page is not blocked for "everyone" group, update the test group permissions if needed
+        JsonNode everyonePerms = null;
```

Comment:
no need to do any checks.
There could be various other permissions which might be set or break the test

Assumption: always create the new permissions for the temporary group